### PR TITLE
Mes-7372: capacitor upgrade

### PR DESCRIPTION
**Pull request by andrewsetterfield**
## Description

Update capacitor to v3

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -16,6 +18,7 @@ import { LogoutBasePageComponent } from '@shared/classes/logout-base-page';
 import { LoadAppVersion, AppResumed, AppSuspended } from '@store/app-info/app-info.actions';
 import { selectLogoutEnabled } from '@store/app-config/app-config.selectors';
 
+
```
Comment (matthew2564) [inline src/app/app.component.ts]: Double line here should be an ESLint failure (I think)

File: src/app/app.component.ts [hunk 2]
```diff
@@ -134,17 +137,11 @@ export class AppComponent extends LogoutBasePageComponent implements OnInit {
   }
 
   configureStatusBar = async (): Promise<void> => {
-    if (Capacitor.isPluginAvailable('StatusBar')) {
```
Comment (matthew2564) [inline src/app/app.component.ts:137]: Does the removal of this check hinder browser development?

File: src/app/app.component.ts (hunk 1 above)
Comment (andrewsetterfield) [reply src/app/app.component.ts]: weird, looks like its waiting to be commited

File: src/app/app.component.ts (hunk 2 above)
Comment (andrewsetterfield) [reply src/app/app.component.ts:137]: it does not. serves and can go through test fine

**Comment by matthew2564** [comment]
Seems to be a number of instances of ```@ionic-native/status-bar``` or ```@ionic-native/status-bar/ngx``` in the code, so looks as if we are mixing the Capacitor/Ionic-native plugins. Wonder if this is an opportunity to remove the ionic-native plugin

**Comment by andrewsetterfield** [comment]
> Seems to be a number of instances of `@ionic-native/status-bar` or `@ionic-native/status-bar/ngx` in the code, so looks as if we are mixing the Capacitor/Ionic-native plugins. Wonder if this is an opportunity to remove the ionic-native plugin

all references updated

**Review by matthew2564 — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -161,9 +158,7 @@
   "cordova": {
     "plugins": {
       "cordova-plugin-whitelist": {},
```
Comment (matthew2564) [inline package.json]: Wonder if we need this line if ```cordova-plugin-whitelist``` has been removed above?
Same for ```cordova-plugin-ionic-webview``` and ```cordova-plugin-ionic-keyboard```.

Can we also check if they can then be removed from ```config.xml```?

Comment (andrewsetterfield) [reply package.json]: good spot, they done seem to be relevent anymore.  will remove and test they still work correctly.

**Review by solomonsimukoko — APPROVED** [review]

**Pull request merged**